assets/src/checkout/pages/BamboraPayPage.tsx

```diff
@@ -82,6 +115,21 @@ const bindEventListeners = (
       setCardErrors(errors => ({ ...errors, [event.field]: "" }));
     }
   });
+  checkout.on("brand", event => {
+    if (event.brand) {
+      if (isCardBrandSupported(gatewayAccount, event.brand)) {
+        setCardErrors(errors => ({ ...errors, [event.field]: "" }));
+      } else {
+        setCardErrors(errors => ({
+          ...errors,
+          [event.field]:
+            event.brand.charAt(0).toUpperCase() +
```

Comment:
you should be using a proper label here, no?

also you can use template literals here instead of string concatenation (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

also, can you please add a `.` to the end of the error message to be consistent with the other Bambora error messages? thanks.

Reply:
done